elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/impl/ElasticsearchIndexManager.java

```diff
@@ -665,7 +665,7 @@ public void performStreamOperation(LuceneWork singleOperation, IndexingMonitor m
 
 	@Override
 	public void optimize() {
-		// TODO Is there such thing for ES?
+		performStreamOperation( OptimizeLuceneWork.INSTANCE, null, false );
```

Comment:
Next time, please, let's keep it separated, or at least in separate commits.

Is there a way to test this?

The issue I was talking about is this one: https://hibernate.atlassian.net/browse/HSEARCH-2092